csharp/language-reference/operators/DelegateOperator.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+
+namespace operators
+{
+    public static class DelegateOperator
+    {
+        public static void Examples()
+        {
+            AnonymousMethod();
+            Lambda();
+            WithoutParameterList();
+        }
+
+        private static void AnonymousMethod()
+        {
+            // <SnippetAnonymousMethod>
+            Action<IEnumerable<int>> display = delegate (IEnumerable<int> s)
+            {
+                Console.WriteLine(string.Join(" ", s));
+            };
+
+            var numbers = new[] { 1, 2, 3 };
+            display(numbers);  // output: 1 2 3
+            // </SnippetAnonymousMethod>
+        }
+
+        private static void Lambda()
+        {
+            // <SnippetLambda>
+            Action<IEnumerable<int>> display = s => Console.WriteLine(string.Join(" ", s));
```

Comment:
I think we need an example that declares multiple parameters, so that readers see that syntax:

```csharp
Func<int, int> sum = (a, b) => a + b;
```

I am pondering a couple other samples to add. I'll comment on those on dotnet/docs#13432

Reply:
I'm fine with the suggestion. Still, we should remember that the article is about the delegate operator. So, main goal is to show that lambdas are more concise. The referenced article about lambdas provides more examples of them.

Reply:
hmm. Very true. I think one example with multiple arguments is wise. That's one bit of syntax I often see questions on. The referenced article is the right location for async lambdas, lambdas with tuples, closures, and so on.

Reply:
@BillWagner I've updated examples to use multiple arguments; those new examples make lambdas look even more better than original examples, I would say. Thanks for the note and suggestion.